src/interfaces/bankAccount.ts

```diff
@@ -1,21 +1,32 @@
-import { IEntity } from './entity'
-import { Vendors } from '../enums/bankAccount'
+import { IEntity } from './entity';
+import { Vendors } from '../enums/bankAccount';
 
 export interface IFlinksContext {
-  loginId: string
-  institution: string
-  accountId?: string
+  loginId: string;
+  institution: string;
+  accountId?: string;
+}
+
+export interface IPaypalContext {
+  payer_id: string;
+  email: string;
+}
+
+export interface IPaypalConnection {
```

Comment:
Why do you need an IPaypal Connection interface?

The only thing you need to do is extend IBankAccount.Context to be IFlinksContext || IPaypalContext, but the IBankAccount vendor will be FLINKS or PAYPAL.

Paypal is a type of Bank account

Reply:
alright